### ISSUE TITLE
Major tag `v1` is crazy old

### ISSUE THREAD

**Issue by lostfields** [booxmedialtd/ws-action-parse-semver#27]
https://github.com/booxmedialtd/ws-action-parse-semver/releases/tag/v1 is a lot older than https://github.com/booxmedialtd/ws-action-parse-semver/releases/tag/v1.4.7

**Comment by axllent** [comment]
You are right, and v1 returns a lot of deprecation notices too (which is fixed in v1.4.7):

> Warning: The `set-output` command is deprecated and will be disabled soon. Please upgrade to using Environment Files. For more information see: https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/
